docs/metrics.md

```diff
@@ -0,0 +1,97 @@
+---
+title: List of metrics exposed by stellar-core
+---
+
+stellar-core uses libmedida for computing metrics, a detailed description can
+be found at http://dln.github.io/medida/
+
+### Counters (`NewCounter`)
+Tracks a value in absolute terms of a base unit.
+
+### Histograms (`NewHistogram`)
+Tracks aggregates (count, min, max, mean, percentiles, etc) for samples
+expressed in arbitrary base unit.
+
+### Timers (`NewTimer`)
+Tracks aggregates (count, min, max, mean, percentiles, etc) of samples expressed in units of time.
+
+### Meters (`NewMeter`)
+Tracks aggregates (count, min, max, mean, etc),  rate (1m, 5m, 15m) for samples
+expressed in base unit.
+
+
+Metric name                       | Type      | Description
+--------------------------------  | --------  | --------------------
+bucket.snap.merge                 | timer     | time to merge two buckets
+bucket.batch.insert               | meter     | number of objects added per batch
+bucket.batch.add                  | timer     | time to add a batch
```

Comment:
I think that names of these two do not pass meaning very well - from the name it is hard to tell which one is insert and which one is add.